tsconfig.json

```diff
@@ -5,15 +5,16 @@
     "allowJs": true,
     "skipLibCheck": true,
     "esModuleInterop": true,
+    "experimentalDecorators": true,
     "allowSyntheticDefaultImports": true,
     "strict": true,
     "forceConsistentCasingInFileNames": true,
-    "module": "esnext",
+    "module": "commonjs",
```

Comment:
Why did you change this one?

Reply:
When I use `esnext` it throws an error ![image](https://user-images.githubusercontent.com/14996067/81148828-d2a73680-8f9e-11ea-83bf-360e02528e7d.png)
so I decided to change it to `commonjs`, so it works

Reply:
So it is quite 'hacky' solution, whenever you introduce a change - you really need to understand why are you doing it. Especially when you are changing the project-wide tsconfig file.

[Read this](https://stackoverflow.com/questions/39883960/what-is-the-typescript-2-0-es2015-way-to-import-assert-from-node-js) or we can switch over to [chai](https://www.chaijs.com/) (actually we already have it in the project)

[This one](https://lihautan.com/javascript-modules/) is a wonderful article about Javascript modules history